test/sol5/kyberNetwork.js

```diff
@@ -786,16 +786,25 @@ contract('KyberNetwork', function(accounts) {
             };
         });
 
-        describe("test getExpectedRate functions with 3 mock reserves, valid rates", async() => {
+        describe("test getExpectedRate functions with rate validating reserves, valid rates", async() => {
             before("setup, add and list reserves", async() => {
                 //init reserves
-                let result = await nwHelper.setupReserves(network, tokens, 3, 0, 0, 0, accounts, admin, operator);
+                let result = await nwHelper.setupReserves(network, tokens.slice(0,3), 0, 0, 0, 9, accounts, admin, operator);
```

Comment:
Because each Apr reserve has 1 token, we need to create 9 reserves for 3 tokens

Reply:
Got it
thanks